front/src/routes/scene/edit-scene/actions/TurnOnOffLightParams.jsx

```diff
@@ -8,36 +8,40 @@ import { ACTIONS } from '../../../../../../server/utils/constants';
 class TurnOnOffLight extends Component {
   getOptions = async () => {
     try {
-      const devices = await this.props.httpClient.get('/api/v1/device', {
+      const deviceFeatures = await this.props.httpClient.get('/api/v1/device', {
         device_feature_category: 'light',
         device_feature_type: 'binary'
       });
-      const deviceOptions = devices.map(device => ({
-        value: device.selector,
-        label: device.name
-      }));
-      await this.setState({ deviceOptions });
+      const deviceFeatureOptions = deviceFeatures
+        .flatMap(device => device.features)
+        .map(deviceFeature => ({
+          value: deviceFeature.selector,
+          label: deviceFeature.name
```

Comment:
I agree with being able to select the feature in this scene action, but the deviceFeature.name is not necessarily good in all cases. 

Some integrations like Philips Hue never exposes the deviceFeature.name so the user doesn't even know it exists.

A mixed approach like we do on other scene actions with `getDeviceFeatureName` function would be better no ? 

Example: https://github.com/GladysAssistant/Gladys/blob/1222d3f9f1c687a1f7e30328abdef86b0c8bf1a4/front/src/routes/scene/edit-scene/actions/DeviceGetValueParams.jsx#L33